src/wallet/db.cpp

```diff
@@ -227,40 +227,12 @@ bool CDB::Recover(const std::string& filename, void *callbackDataIn, bool (*reco
 
 bool CDB::VerifyEnvironment(const std::string& walletFile, const fs::path& dataDir, std::string& errorStr)
 {
-    LogPrintf("Using BerkeleyDB version %s\n", DbEnv::version(0, 0, 0));
```

Comment:
You're right. This (and `CWalletDB::VerifyEnvironment()`) are now entirely vestigial and are not called. Both can be removed.